### ISSUE TITLE
Amazon Echo Hub, Amazon Echo Device on Home Assistant using Node Red (Devices Not Discovered)

### ISSUE THREAD

**Issue by esovetsky** [datech/node-red-contrib-amazon-echo#192]
##### SUMMARY
I am using Node-Red 14.0.0 with the Home Assistant 2022.12.8.   I have placed the "Amazon Echo Hub" node on the page with default port 80 and device discovery checked.  I placed "Amazon Echo Device" on the page and gave it a unique name.  Then I placed a "Service Call" node and link all three together.  I asked Alexa to discover, and nothing was discovered.   The "Amazon Echo Device" shows a green dot below stating "online".   I did hang a debug off the "Amazon Echo Hub" and no output is presented in the debug window.   I confirmed both with wireless and wired via a browser that (http://192.168.10.175:80/api/description.xml) yields information.  Home Assistant is running on a Dell computer (Home Assistant 2022.12.8, Supervisor 2022.12.1, Operating System 9.4, Frontend 20221213.1 - latest). Any help in resolving this issue is appreciated.

##### ECHO DEVICES
* Echo Dot 2n generation
* Echo Dot 3rd generation
* Echo Plus 2nd generation
* Amazon Echo Flex 
* Echo Show 5"

##### CONFIGURATION

```paste below
[{"id":"7144f35f176b74bd","type":"tab","label":"Flow 3","disabled":false,"info":"","env":[]},{"id":"de3d6088108d3b7d","type":"amazon-echo-hub","z":"7144f35f176b74bd","port":"80","processinput":0,"discovery":true,"x":110,"y":100,"wires":[["cda93f7ee24270d5","e7be1cd46bbaed75"]]},{"id":"cda93f7ee24270d5","type":"amazon-echo-device","z":"7144f35f176b74bd","name":"Test Light","topic":"","x":300,"y":100,"wires":[["a3c552b15a6ba4f9"]]},{"id":"a3c552b15a6ba4f9","type":"api-call-service","z":"7144f35f176b74bd","name":"Turn On","server":"95701f61.2d3a5","version":5,"debugenabled":false,"domain":"switch","service":"turn_off","areaId":[],"deviceId":["c238617fd5a4f29d8345a23937762bbb"],"entityId":[],"data":"","dataType":"jsonata","mergeContext":"","mustacheAltTags":false,"outputProperties":[],"queue":"none","x":460,"y":100,"wires":[[]]},{"id":"e7be1cd46bbaed75","type":"debug","z":"7144f35f176b74bd","name":"debug 3","active":true,"tosidebar":true,"console":false,"tostatus":false,"complete":"false","statusVal":"","statusType":"auto","x":300,"y":200,"wires":[]},{"id":"95701f61.2d3a5","type":"server","name":"Home Assistant","addon":true}]```

##### CONSOLE OUTPUT
<!--- Paste Node-RED console output between quotes -->
```paste below

**Comment by harvybob** [comment]
Not sure if it makes a difference, but I put the DEBUG after the light 
![image](https://user-images.githubusercontent.com/17744328/210313678-c057a0a7-20ed-42e2-8aaf-d24f36970093.png)
Also, have you enabled the Philips hue Integration on HA or do you already have it? (I think it uses that as a dummy interface!)

**Comment by esovetsky** [comment]
harvybob,

So, I tried added the Philips Hue Intergration and then tired to configure.  I get the following:
<img width="213" alt="image" src="https://user-images.githubusercontent.com/119077953/210434639-868b35b3-44b1-4abc-9b07-8f940653677b.png"> 
<img width="401" alt="image" src="https://user-images.githubusercontent.com/119077953/210434753-edd343bf-5d07-4708-ada9-9ab70e2212c1.png">

Also, I did put the debug, but I don't get anything hitting the "Amazon Echo Hub"

<img width="410" alt="image" src="https://user-images.githubusercontent.com/119077953/210434935-c8c541af-308d-47e6-a169-87eb2be87823.png">

I have a feeling it has to do with making Port 80 work.  But just not sure.
